tirgei/stack-overflow-lite
# #162563775 A user can create an account

#### What does this PR do?
This Pull Request creates a register page through which a new user can sign up on the platform

#### Description of Task to be completed?
The tasks for this PR is:
- Create a form for user to input details using HTML
- Style the details form using CSS

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/stack-overflow-lite.git ```

CD into the project folder on your machine
```$ cd stack-overflow-lite ```

Check out the branch for this PR
``` $ git branch ft-user-register-162563775 ```

Open the `register.html` file on your browser

### Pivotal Tracker story associated with this PR
From [StackOverflow-lite](https://www.pivotaltracker.com/n/projects/2231260) project, story: #162563775

#### Screenshot
![register-stack-overflow-lite](https://user-images.githubusercontent.com/17829576/49869718-47198c80-fe22-11e8-855f-fa202d95b5a5.png)
